Source/KerbalCombatSystems/MissileGuidance.cs

```diff
@@ -34,14 +34,14 @@ public class ModuleMissile : ModuleWeapon
         public float maxAcceleration;
         private Vector3 rcs;
         private float targetSize;
+        private Vector3 propulsionVector;
 
         // Components
 
         private KCSFlightController fc;
         private ModuleDecouplerDesignate seperator;
         private ModuleWeaponController controller;
         private List<ModuleEngines> engines;
-        private List<ModuleRCSFX> rcsList;
```

Comment:
move the rcs list storage back to here so that it can be recalled during flight to update